lib/pdk/module/metadata.rb

```diff
@@ -54,12 +54,16 @@ class Metadata
           'operatingsystem'        => 'windows',
           'operatingsystemrelease' => ['2008 R2', '2012 R2', '10'],
         },
+        'Archlinux' => {
```

Comment:
The user friendly name appears to be "Arch Linux" so we should use that as the key here.

Reply:
updated